railties/lib/rails/generators/rails/plugin/plugin_generator.rb

```diff
@@ -171,12 +181,12 @@ def stylesheets
       end
     end
 
-    def bin(force = false)
-      bin_file = engine? ? "bin/rails.tt" : "bin/test.tt"
-      template bin_file, force: force do |content|
+    def bin
```

Comment:
Was the `force` argument unused?

Reply:
It was always set to true but I changed this to match the app generator as well.